src/RingCentralCall.ts

```diff
@@ -246,63 +298,118 @@ export class RingCentralCall extends EventEmitter {
-      }
-    });
-  }
+    this._sdk
+      .platform()
+      .loggedIn()
+      .then((loggedIn) => {
+        if (loggedIn) {
+          this._subscription
+            .reset()
+            .setEventFilters(this._subscriotionEventFilters)
+            .register();
+        }
+      });
+  };
 
   _onSubscriptionRenewFail = () => {
     this._callControlNotificationReady = false;
     // @ts-ignore
     this.emit(events.CALL_CONTROL_NOTIFICATION_ERROR);
-    this._sdk.platform().loggedIn().then((loggedIn) => {
-      if (loggedIn) {
-        this._subscription.reset().setEventFilters(this._subscriotionEventFilters).register();
-      }
-    });
-  }
+    this._sdk
+      .platform()
+      .loggedIn()
+      .then((loggedIn) => {
+        if (loggedIn) {
+          this._subscription
+            .reset()
+            .setEventFilters(this._subscriotionEventFilters)
+            .register();
+        }
+      });
+  };
 
   _onSubscriptionAutomaticRenewError = () => {
     this._subscription.resubscribe();
-  }
+  };
 
   _clearSubscriptionHander() {
     if (!this._subscription) {
       return;
     }
-    this._subscription.removeListener(this._subscription.events.subscribeSuccess, this._onSubscriptionCreateSuccess);
-    this._subscription.removeListener(this._subscription.events.renewSuccess, this._onSubscriptionCreateSuccess);
-    this._subscription.removeListener(this._subscription.events.notification, this.onNotificationEvent);
-    this._subscription.removeListener(this._subscription.events.renewError, this._onSubscriptionRenewFail);
-    this._subscription.removeListener(this._subscription.events.automaticRenewError, this._onSubscriptionAutomaticRenewError);
-    this._subscription.removeListener(this._subscription.events.subscribeError, this._onSubscriptionCreateError);
+    this._subscription.removeListener(
+      this._subscription.events.subscribeSuccess,
+      this._onSubscriptionCreateSuccess
+    );
+    this._subscription.removeListener(
+      this._subscription.events.renewSuccess,
+      this._onSubscriptionCreateSuccess
+    );
+    this._subscription.removeListener(
+      this._subscription.events.notification,
+      this.onNotificationEvent
+    );
+    this._subscription.removeListener(
+      this._subscription.events.renewError,
+      this._onSubscriptionRenewFail
+    );
+    this._subscription.removeListener(
+      this._subscription.events.automaticRenewError,
+      this._onSubscriptionAutomaticRenewError
+    );
+    this._subscription.removeListener(
+      this._subscription.events.subscribeError,
+      this._onSubscriptionCreateError
+    );
     this._subscription = null;
   }
 
   onNotificationEvent = (msg) => {
-    this._callControl.onNotificationEvent(msg)
+    this._callControl.onNotificationEvent(msg);
+  };
+
+  loadSessions = async (sessions) => {
+    await this._callControl.loadSessions(sessions);
+    this._sessions = this._callControl.sessions as Session[];
```

Comment:
this._callControl.sessions.forEach((session) => {
  this._onNewTelephonySession(session, true)
});